src/parser/jobs/war/modules/StormsEye.tsx

```diff
@@ -0,0 +1,86 @@
+import {Trans} from '@lingui/react'
+import React from 'react'
+
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import {BuffEvent} from 'fflogs'
+import Module, {dependency} from 'parser/core/Module'
+import Checklist, {Rule, Requirement} from 'parser/core/modules/Checklist'
+import Combatants from 'parser/core/modules/Combatants'
+import {EntityStatuses} from 'parser/core/modules/EntityStatuses'
+import {Invulnerability} from 'parser/core/modules/Invulnerability'
+import Suggestions, {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+
+const STORMS_EYE_BUFFER = 7000
+
+export default class StormsEye extends Module {
+	static handle = 'stormseye'
+
+	@dependency private checklist!: Checklist
+	@dependency private combatants!: Combatants
+	@dependency private entityStatuses!: EntityStatuses
+	@dependency private invuln!: Invulnerability
+	@dependency private suggestions!: Suggestions
+
+	private earlyEyes: number = 0
+	private totalEyes: number = 0
+	private lastRefresh: number = this.parser.fight.start_time
+
+	protected init(): void {
+		this.addEventHook('applybuff', {by: 'player', abilityId: STATUSES.STORMS_EYE.id}, this.onGain)
+		this.addEventHook('refreshbuff', {by: 'player', abilityId:  STATUSES.STORMS_EYE.id}, this.onGain)
+		this.addEventHook('complete', this.onComplete)
+	}
+
+	onGain(event: BuffEvent): void {
+		this.lastRefresh = event.timestamp
+		this.totalEyes++
+
+		if (this.totalEyes < 2) {
+			return
+		}
+
+		if (this.parser.patch.before('5.3')) {
+			const remaining = event.timestamp - this.lastRefresh
+			if (remaining < STATUSES.STORMS_EYE.duration - STORMS_EYE_BUFFER) {
```

Comment:
Your new layered data updates for 5.3 aren't going to work without using `this.data.actions.STORMS_EYE.duration` I think!

Reply:
Okay technically they will work since these will be grabbing the StatusRoot values, which is the original 30s, but STILL

Reply:
I'm pretty sure it uses the final merged list, since I see this working fine in MNK which does a similar thing. Prob should update it to have nicer typing tho, just gotta fill array objects like what you do in Horoscope or whatever.

Reply:
If it works then I'll just give it a pass and we can go back and conform everything later